h2/src/main/org/h2/schema/Schema.java

```diff
@@ -63,9 +64,10 @@
      * @param owner the owner of the schema
      * @param system if this is a system schema (such a schema can not be
      *            dropped)
+     * @param tableEngineParams Table engine params to use when none are present in CREATE TABLE.
      */
     public Schema(Database database, int id, String schemaName, User owner,
-            boolean system) {
+            boolean system, ArrayList<String> tableEngineParams) {
```

Comment:
We have setter method in `CreateSchema` AST class, not in `Schema`, it's a final field here. Should I rework `Schema` to use setter instead of a final field?